charts/rancher-backup/files/default-resourceset-contents/main.yaml

```
@@ -18,6 +18,9 @@
 - apiVersion: "v1"
   kindsRegexp: "^serviceaccounts$"
   namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
+- apiVersion: "v1"
+  kindsRegexp: "^services$"
+  namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
```

I think we should select only the webhook's one by calling out its name `rancher-webhook`. 
 ```suggestion
   namespaceRegexp: "^cattle-|^p-|^c-|^local$|^user-|^u-"
   resourceNames:
     - "rancher-webhook"
```

---

Hmm, probably a good idea. However, we should keep deployments and serviceaccounts the same.

---

it is a good point.  
It is ok that we just backup all services under the target namespaces if that is how we select other resources